### PR TITLE
docs/digitalocean: Add image data source to sidebar

### PR DESCRIPTION
**Pull request by radeksimko**
Ref #13787

### CODE REVIEW

File: website/source/layouts/digitalocean.erb [hunk 1]
```diff
@@ -11,6 +11,15 @@
                 </li>
 
         <li<%= sidebar_current("docs-do-resource") %>>
```
Comment (tombuildsstuff) [inline website/source/layouts/digitalocean.erb]: Given this is a data source - I think this should be `docs-do-datasource`?

File: website/source/layouts/digitalocean.erb [hunk 2]
```diff
@@ -11,6 +11,15 @@
...
+        <a href="#">Data Sources</a>
+                <ul class="nav nav-visible">
+                    <li<%= sidebar_current("docs-do-resource-image") %>>
```
Comment (tombuildsstuff) [inline website/source/layouts/digitalocean.erb]: Given this is a data source, should this be `docs-do-datasource-image`?

Comment (radeksimko) [inline website/source/layouts/digitalocean.erb]: ah, good point, will fix that

**Comment by radeksimko** [comment]
@tombuildsstuff fixed.

**Pull request merged**